### DISCUSSION TITLE
API queries and mutations to play radio messages in live timing

### DISCUSSION THREAD

**Discussion by RobSpectre** [f1multiviewer/issue-tracker#580] [Feature requests]
Would love some new queries and mutations to play radio messages from the live timing window.

The initial use case I'd really like is to press a button on the streamdeck, mute the broadcast and then play the latest radio message that came up.

**Comment by f1multiviewer** [comment]
Hmmm that's an interesting use-case. I'll think about it, I'm not quite sure how that would work, but I guess I could make it work if I wire it up correctly.

I'm planning some changes to live timing that may make this a little more difficult (as the goal is to not require the live timing window anymore), but maybe I can create an invisible window just for playing these back, or play them back in the main window, or in a player. I'm not sure. Will think about it, but might take some time given my current priorities :)

**Comment by RobSpectre** [comment]
Thanks for the swift response, as always.

**Comment by LapsTimeOFF** [comment]
Closing as issue is stale.
